requirements/base.txt

```diff
@@ -1,9 +1,11 @@
-praw==4.4.0
+praw==5.0.1
```

Comment:
I installed it and nothing seemed to explode; this is one of those "keep a close eye on it" things.